mdo/.htaccess

```
@@ -37,7 +37,7 @@ RewriteRule ^(\w+)/(\w+)$ https://huanyu-li.github.io/mdo/$1/1.0/index.html [R=3
 
 
 RewriteCond %{HTTP_ACCEPT} ^.*application/rdf\+xml.*
-RewriteRule ^(\w+)$ https://huanyu-li.github.io/mdo/$1/1.0/mdo-$1.owl [R=303,NE,L]
+RewriteRule ^(\w+)/$ https://huanyu-li.github.io/mdo/$1/1.0/mdo-$1.owl [R=303,NE,L]
```

You could make the trailing slash optional:
```suggestion
RewriteRule ^(\w+)/?$ https://huanyu-li.github.io/mdo/$1/1.0/mdo-$1.owl [R=303,NE,L]
```

---

Thanks.